sdk/textanalytics/cognitiveservices-textanalytics/src/textAnalyticsClient.ts

```diff
@@ -57,30 +57,49 @@ export interface TextAnalyticsClientOptions extends PipelineOptions {
   defaultLanguage?: string;
 }
 
+/**
+ * Options common to all text analytics operations.
+ */
 export interface TextAnalyticsOperationOptions extends OperationOptions {
   /**
-   * (Optional) This value indicates which model will be used for scoring. If a model-version is
+   * This value indicates which model will be used for scoring. If a model-version is
```

Comment:
nit: you've been following the .NET docstrings convention to start a property docstring with Gets (or "Gets or sets") until here.  Not sure if you want to continue to follow that convention throughout... ?

Reply:
Ah, an unintentional result of my copy paste villainy. I think I'm going to change this away from 'gets', since the other JS libs don't follow this convention and it might confuse consumers who would think these are actual getters instead of simple properties.